src/dotnet/UI.Blazor/Components/Account/AccountPresenceIndicator.razor

```diff
@@ -0,0 +1,43 @@
+﻿@namespace ActualChat.UI.Blazor.Components
+@using ActualChat.Users
+@inherits ComputedStateComponent<Presence>
+
+@{
+    var presence = State.Value;
+    if (presence is Presence.Unknown or Presence.Offline)
+        return;
+}
+
+@PresenceIndicator.Render(presence)
+
+@code {
+    [Inject] private IUserPresences UserPresences { get; init; } = null!;
+
+    protected UserId UserId { get; private set; }
+
+    [Parameter, EditorRequired] public string UserSid { get; set; } = "";
+
+    protected override void OnParametersSet()
+        => UserId = new UserId(UserSid);
+
+    protected override ComputedState<Presence>.Options GetStateOptions() {
+        if (UserId.IsNone)
+            return new ComputedState<Presence>.Options {
+                InitialValue = Presence.Unknown,
+                Category = GetStateCategory(),
+            };
+        var presenceComputed = Computed.GetExisting(() => UserPresences.Get(UserId, default));
+        var presence = presenceComputed?.IsConsistent() == true &&  presenceComputed.HasValue ? presenceComputed.Value : Presence.Unknown;
+
+        return new ComputedState<Presence>.Options {
+            InitialValue = presence,
+            Category = GetStateCategory(),
+        };
+    }
+
+    protected override async Task<Presence> ComputeState(CancellationToken cancellationToken) {
+        if (UserId.IsNone)
+            return Presence.Unknown;
+        return await UserPresences.Get(UserId, cancellationToken);
```

Comment:
Either an empty line is missing or you need to convert it into a ternary "?:".

Reply:
Any `return` implies an empty line _after_, unless:
- It's the last line of a statement block (i.e. it follows with `}`)
- It's a `return` inside a sequence of `if (...) return` - in this case only the last return in such a sequence needs an empty line. Sorta fit here, but I'd rather use `?:`.